pilot/pkg/security/authz/matcher/header.go

```diff
@@ -19,10 +19,28 @@ import (
 
 	routepb "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
 	matcherpb "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+
+	"istio.io/istio/pilot/pkg/features"
 )
 
 // HeaderMatcher converts a key, value string pair to a corresponding HeaderMatcher.
 func HeaderMatcher(k, v string) *routepb.HeaderMatcher {
+	// If environment variable AUTHZ_ENABLE_REGEX is set to ture, prefix "regex:"
```

Comment:
I think you can just remove these comments, the code is pretty clear about this.